src/System.Data.Common/src/System/Data/Common/DbDataReader.cs

```diff
@@ -193,7 +193,7 @@ public virtual Task<bool> IsDBNullAsync(int ordinal, CancellationToken cancellat
             {
                 try
                 {
-                    return IsDBNull(ordinal) ? ADP.s_trueTask : ADP.s_falseTask;
+                    return IsDBNull(ordinal) ? ADP.TrueTask : ADP.FalseTask;
```

Comment:
because in https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs#L58 they are defined like properties (in Common - static fields with inline initializations).